mist-api/src/lib/RouteHelper.js

```diff
@@ -202,7 +229,64 @@ class RouteHelper {
 		})
 	}
 
+	findByRawQuery_(sequelize, res, countRows = false) {
+		let criteria = res.locals.criteria;
```

Comment:
We should avoid building raw queries like this wherever possible as it is riddled with opportunity for SQL injection and other malicious manipulation (both known and unknown). Raw queries enable us to do anything; however, they need to be used and implemented extremely carefully. Leaving the "safer" world of the sequelize ORM requires tons more caution.

I would like to find an alternative strategy that accomplishes the same goal with less exposure.